content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,374 @@
+  to search for on *X*, like `#WoodWorkingExpo24`, for example.
+
+  Next, determine the :guilabel:`Type of tweets` Odoo should showcase with that predetermined
+  criteria. The choices in this field are: :guilabel:`Recent`, :guilabel:`Popular`, or
+  :guilabel:`Mixed`.
+
+  Users also have the option to add a brief :guilabel:`Description` to the wall, as well.
+
+  Lastly, the greyed-out, non-modifiable :guilabel:`Website URL` field will autopopulate with the
+  full URL needed to access the document through the event website.
+
+  An image can also be added to the wall by clicking the :guilabel:`🖉 (pencil)` icon that appears
+  when the cursor hovers over the :guilabel:`camera` box in the upper-right corner of the pop-up
+  window. Then, from the resulting file explorer window, select the desired image to be added to
+  the wall.
+
+  This field **only** appears on the event form if the *Social Marketing* app is installed, and an
+  *X* account has been added as a stream on the application.
+- :guilabel:`Template`: choose a pre-configured event template from the resulting drop-down menu.
+
+   Or, create a new one directly from this field, by typing in the name of the new template, and
+   selecting either: :guilabel:`Create` (which creates the template, and can be edited later) or
+   :guilabel:`Create and edit...` (which creates the template, and reveals a separate template page
+   to configure the template in greater detail).
+- :guilabel:`Tags`: add any corresponding tags to briefly describe the event (e.g. `Online`,
+  `Conference`, etc.). Multiple tags can be added per event.
+- :guilabel:`Organizer`: designate the organizer of the event.
+- :guilabel:`Responsible`: designate a user in the database to be responsible for this event.
+- :guilabel:`Company`: designate which company in the database to which this event is related. This
+  field **only** appears if working in a multi-company environment. If left blank, the event will
+  relate to all companies in the database.
+- :guilabel:`Website`: choose to restrict the publishing of this event to a specific website. If
+  this field is left blank, the event can be published on *all* websites in the database.
+- :guilabel:`Venue`: enter the pertinent event venue details in this field.
+- :guilabel:`Exhibition Map`: if desired, click the :guilabel:`Upload your file` button to Upload
+  a map of the event venue.
+- :guilabel:`Limit Registrations`: if this checkbox is ticked, a limit to the amount of
+  registrations is added to the event, and that desired limit amount **must** be entered in the
+  blank field before :guilabel:`Attendees`.
+- :guilabel:`Badge Dimension`: select a desired dimension for event badges. The options are:
+  :guilabel:`A4 foldable`, :guilabel:`A6`, or :guilabel:`4 per sheet`.
+- :guilabel:`Badge Background`: if desired, click the :guilabel:`Upload your file` button to upload
+  a custom background for event badges.
```

Comment:
```suggestion
- :guilabel:`Badge Background`: if desired, click the :guilabel:`Upload your file` button to upload
  a custom background for event badges.
```
Is there a recommended format- what type of images will look best? a vector? a full color image?

Reply:
It is my understanding that we are merely supposed to instruct users how to use the software, and leave all recommendations out. Kinda leave it up to them, ya know? Plus, every instance could be different, depending on image size, quality, etc. -- so, with that said, I think it would be best to ignore such a recommendation, and leave it up to the user(s) to decide.